api/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -150,27 +150,6 @@
    */
   public abstract T overrideAuthority(String authority);
 
-  /**
-   * Use of a plaintext connection to the server. By default a secure connection mechanism
-   * such as TLS will be used.
-   *
-   * <p>Should only be used for testing or for APIs where the use of such API or the data
-   * exchanged is not sensitive.
-   *
-   * @param skipNegotiation @{code true} if there is a priori knowledge that the endpoint supports
-   *                        plaintext, {@code false} if plaintext use must be negotiated.
-   * @deprecated Use {@link #usePlaintext()} instead.
-   *
-   * @throws UnsupportedOperationException if plaintext mode is not supported.
-   * @return this
-   * @since 1.0.0
-   */
-  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1772")
-  @Deprecated
-  public T usePlaintext(boolean skipNegotiation) {
```

Comment:
There's plenty of existing users. We should leave this in-place for a good while (at least a year).

Reply:
We have already deprecated it for one and half year, still need a year?

Reply:
Sure. Do you mind if I also log a WARNING in the method?

Reply:
Meh. They would already get a warning during compilation. I'd rather us not think about it much and just leave it setting there for a while.

Reply:
Done.

Reply:
Sorry, I misunderstood what you said about a year and a half. Yes, that is plenty. We can remove this. Although we've also found some users that need to be migrated internally. We'll remove it once they are migrated.